src/init.tbasicu

```diff
@@ -1,18 +1,61 @@
+#Region "TBGL Global Settings"
+
+tbgl_UseTexturing %TRUE
+tbgl_UseAlphaTest %TRUE
+
+#EndRegion
+
+#Region "title"
+
+%SIDE = 4
+
+%CTR = 50
+%CTG = 0
+%CTB = 0
+
+%CSR = 0
+%CSG = 0
+%CSB = 50
+
+%MAXBULLES = 30
+%GROUND = 5
+%BOOST = 600
+%TIMEBOOST = 400
+%TIMETITLE = 650
+
+$TITLEPNG = "\media\BTB1024.png"
+$TITLESPRITEPNG = "\media\bulle512.png"
+
+
+type tBulles
+  X     as double
+  Y     as double
+  Z     as double
+  Size  as double
+  Speed as double
+  Fade  as double
+  R     as double
```

Comment:
I think R, G, B should be BYTE

Reply:
right you are

Reply:
But when I set to BYTE here and UBYTE in the #compiled section, thinBasic crashes. 
Anyway, I've removed them in next version.

Reply:
If you want to investigate this freeBasic related question...
If not, we can resolve this conversation.